### PR TITLE
[ContactStructuralMechanicsApplication] Tests relative paths

### PR DESCRIPTION
**Pull request by loumalouomega**
Fixes a relative path

### CODE REVIEW

**Review by AlejandroCornejo — APPROVED** [review]
ok!

**Comment by loumalouomega** [comment]
> ok!

Thanks

File: applications/ContactStructuralMechanicsApplication/tests/test_ContactStructuralMechanicsApplication.py [hunk 1]
```diff
@@ -18,14 +18,18 @@
     missing_application = re.search(r'''.*'KratosMultiphysics\.(.*)'.*''',
                                     '{0}'.format(e)).group(1)
 
+import os
+import sys
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+sys.path.insert(0, GetFilePath('../../../kratos/tests/'))
```
Comment (philbucher) [inline applications/ContactStructuralMechanicsApplication/tests/test_ContactStructuralMechanicsApplication.py:25]: inserting things on the path is potentially dangerous, maybe you could use the KratosPaths instead?

Btw does this fix the nightly test-failure?

Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/tests/test_ContactStructuralMechanicsApplication.py:25]: Yep

Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/tests/test_ContactStructuralMechanicsApplication.py:25]: I don't know how to use KratosPaths, I tried yesterday, and this directly worked

**Pull request merged**